nri/css/style.css

```
@@ -0,0 +1,367 @@
+@charset "UTF-8";
+
+/* 共通部分
+-------------------- */
+
+html {
+    font-size: 100%;
+    font-feature-settings: "palt"
+}
+body{
+    font-family: 'Noto Sans JP',sans-serif!important;
+}
+a{
+    text-decoration: none;
+}
+img{
+    max-width: 100%;
+}
+li{
+    list-style-type: none;
+}
+/*ヘッダー
+-------------- */
+header{
+    height: 65px;
+    width: 100%;
+    background-color: white;
+    position: fixed;
+    z-index: 1;
+}
+nav{
+    margin: 0 auto;
+    max-width: 1600px;
+    margin: 0 auto;
+}
+.header-nav{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.header-logo{
+    height: 65px;
+}
+.main-nav{
+    display: flex;
+    font-size: 14px;
+    margin-left: 13.5rem;
+    font-weight: bold;
+}
+.main-nav li{
+    margin-right:15px;
+}
+.main-nav a{
+    color: unset;
+}
+.main-nav a:hover{
+    border-bottom: 2px solid #0c419a;
+    padding-bottom: 18px;
+    color:#0c419a;
+}
+.sub-nav{
+    display: flex;
+    margin-right: 75px;
+}
+.sub-nav a:hover{
+    opacity: 0.5;
+    cursor: pointer;
+}
+.sub-nav li{
+    margin-left: 40px;
+}
+/*本文
+---------------------*/
+.article{
+    width: 750px;
+    margin: 0 auto;
+    }
+.header-index{
+    padding: 65px 0px 25px 300px;
+    display: flex;
+    font-size: 12px;
+    color: gray;
+}
+.header-index a{
+    color: gray;
+}
+.header-index a:after{
+    content: ">";
+    margin: 0 2px;
+}
+.article-title{
+    height: 250px;
+    background-image: url(../images/h1_whats_2.jpg);
+    background-size: cover;
+    background-position: center;
+    color: white;
+}
+.title{
+    max-width: 1000px;
+    margin: 0 auto;
+    padding-top: 40px;
+}
+.icon-item img{
+    height: 25px;
+}
+.ico{
```

他の要素はiconを使っているので、こちらも揃えましょう。

```suggestion
.icon {
```